components/global/ArticleCard.tsx

```diff
@@ -85,7 +78,7 @@ border-color:  ${props => props.isPublished ?  !props.bookmarked  ?'#4EC700 !imp
 
 
 /* article state: draft, review, pubished, complete*/
```

Comment:
The spacing for the right side is way off. It looks like the image container has moved to the left side of the like bar when it should be above the title
![Screen Shot 2020-07-09 at 11 06 32 AM (2)](https://user-images.githubusercontent.com/35946905/87074912-6b8f4380-c1d4-11ea-8427-c78e6f71d3cb.png)

Reply:
Okay will check this out! I am not seeing this spacing in my laptop actually